### PR TITLE
Adds New Firearms to the Cargo catalogue at a premium

### PR DESCRIPTION
**Pull request by CCC23**
## About The Pull Request

This PR adds the accelerator laser cannon and scatter laser gun to the cargo catalogue for the price of 12k 

## Why It's Good For The Game
More guns being able to be accessible in game can allow people to fight in more interesting ways and letting these guns rot unused discourages people from using already available laser guns as they seem boring this pr is made to let the fancy among us use some nicer more specialized weapons

## Changelog

:cl:
add: Added 2 "New" laser guns to the cargo catalogue
/:cl:

### CODE REVIEW

**Comment by CCC23** [comment]
whoops opened to soon

File: code/modules/cargo/packs/security.dm [hunk 1]
```diff
@@ -17,6 +17,16 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/Specialist
+	name = "Specialist Laser Pack
```
Comment (Kylerace) [inline code/modules/cargo/packs/security.dm:21]: unclosed "

**Comment by CCC23** [comment]
it might be trying to merge the wrong thing lemmie figure this out

File: code/modules/cargo/packs/security.dm [hunk 2]
```diff
@@ -17,6 +17,16 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/Specialist
```
Comment (Kylerace) [inline code/modules/cargo/packs/security.dm:20]: typepaths are in lower case only

**Review by Kylerace — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs/security.dm (hunk 2 above)
Comment (CCC23) [reply code/modules/cargo/packs/security.dm:20]: thanks didnt notice this at first

**Comment by CCC23** [comment]
damn it what I did I do this time

**Comment by CCC23** [comment]
forgot to update that

**Comment by Livrah** [comment]
>"fight in more interesting ways"
>kills the Blob from a mile away in interesting way

**Comment by CCC23** [comment]
> > "fight in more interesting ways"
> > kills the Blob from a mile away in interesting way

if your willing to spend the equivalent of 35 laser guns to get 1 gun then go ahead

File: code/modules/cargo/packs/security.dm [hunk 3]
```diff
@@ -17,6 +17,15 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/specialist
+	name = "Specialist Laser Pack"
+	desc = "Contains a set of Nanotransets more specialized laser weaponry."
+	cost = CARGO_CRATE_VALUE * 60
+	contains = list(
+		/obj/item/gun/energy/lasercannon
```
Comment (Kylerace) [inline code/modules/cargo/packs/security.dm:25]:
```suggestion
		/obj/item/gun/energy/lasercannon,
```

**Comment by Serxule** [comment]
isnt the accelerator already purchasable as a goodie for fairly cheap

**Comment by CCC23** [comment]
> isnt the accelerator already purchasable as a goodie for fairly cheap

no thats the 38 special the accelerator is only avaliable right now through 1 ruin in deep space in a turret which requires you to get a firing pin

**Comment by Serxule** [comment]
> > isnt the accelerator already purchasable as a goodie for fairly cheap
> 
> no thats the 38 special the accelerator is only avaliable right now through 1 ruin in deep space in a turret which requires you to get a firing pin

ok 2 things
1. i was thinking of the hellfire laser gun
2. how do you think i would mistake a ballistic revolver for a laser gun

**Comment by CCC23** [comment]
> > > isnt the accelerator already purchasable as a goodie for fairly cheap
> > 
> > 
> > no thats the 38 special the accelerator is only avaliable right now through 1 ruin in deep space in a turret which requires you to get a firing pin
> 
> ok 2 things
> 
> 1. i was thinking of the hellfire laser gun
> 2. how do you think i would mistake a ballistic revolver for a laser gun

the other expensive goodie weapon is the revolver its the only other thing that costs way to much

File: code/modules/cargo/packs/security.dm [hunk 4]
```diff
@@ -17,6 +17,15 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/specialist
+	name = "Specialist Laser Pack"
+	desc = "Contains a set of Nanotransets more specialized laser weaponry."
+	cost = CARGO_CRATE_VALUE * 60
+	contains = list(
+		/obj/item/gun/energy/lasercannon,
+		/obj/item/gun/energy/laser/scatter
```
Comment (dragomagol) [inline code/modules/cargo/packs/security.dm:26]:
```suggestion
		/obj/item/gun/energy/laser/scatter,
```

**Comment by CCC23** [comment]
ah forgot that thanks guys

**Comment by improvedname** [comment]
atleast it isnt the accelator ;')

**Comment by CCC23** [comment]
> accelator is really a finicky gun to put in cargo because the accelator encourages critting off vision which isn't a very fun gameplay loop

and 12k encorages you to REALLY want the gun

**Comment by Serxule** [comment]
> > accelator is really a finicky gun to put in cargo because the accelator encourages critting off vision which isn't a very fun gameplay loop
> 
> and 12k encorages you to REALLY want the gun

not really since the mosins a lot cheaper and way stronger

**Comment by improvedname** [comment]
There is a reason why this gun is orginally locked behind research you'll see the results of whats gonna happen when this gets trough accelator gun can really rack up in damage

**Comment by Serxule** [comment]
well yes because if you spam it down hallways its strong, otherwise it doesnt really have a use, its basically just a "specifically fuck the blob" weapon

File: code/modules/cargo/packs/security.dm [hunk 5]
```diff
@@ -17,6 +17,15 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/specialist
+	name = "Specialist Laser Pack"
+	desc = "Contains a set of Nanotransets more specialized laser weaponry."
```
Comment (Zonespace27) [inline code/modules/cargo/packs/security.dm:22]:
```suggestion
	desc = "Contains a set of Nanotrasen's more specialized laser weaponry."
```

File: code/modules/cargo/packs/security.dm [hunk 6]
```diff
@@ -17,6 +17,15 @@
 					/obj/item/ammo_box/c38/iceblox,
 				)
 	crate_name = "ammo crate"
+/datum/supply_pack/security/armory/specialist
```
Comment (Zonespace27) [inline code/modules/cargo/packs/security.dm:20]: space between line 19 and line 20

**Comment by improvedname** [comment]
well we'll see how this gonna turn out

**Comment by Fikou** [comment]
> if your willing to spend the equivalent of 35 laser guns to get 1 gun then go ahead

why would i buy it then

**Comment by SkeletalElite** [comment]
> why would i buy it then

because buying guns isn't always about arming as many people as possible.
In response to a blob, you should arm as many people as possible. He was simply pointing out this isn't indirectly nerfing blobs because when it comes specifically to dealing with them this would not be the most effective use of money. But that doesn't mean they are useless or just fun to buy sometime just because.

**Comment by improvedname** [comment]
(me setting up reflectors to oneshot the blob)

**Comment by Fikou** [comment]
> > why would i buy it then
> 
> because buying guns isn't always about arming as many people as possible. In response to a blob, you should arm as many people as possible. He was simply pointing out this isn't indirectly nerfing blobs because when it comes specifically to dealing with them this would not be the most effective use of money. But that doesn't mean they are useless or just fun to buy sometime just because.

idk i think id rather just buy a fun shuttle than spend 12000 on 2 average guns

**Pull request closed without merging**